modules/atomic-widgets/props-resolver/transformer-base.php

```diff
@@ -6,5 +6,13 @@
 }
 
 abstract class Transformer_Base {
-	abstract public function transform( $value );
+	abstract public function transform( $value, $key );
+
+	protected function multi( $value ) {
```

Comment:
Remove from the base transformer and create it inside of some kind of util `Multi_Props` then it can have:
`generate`: create the asoc array
`is`: check if the value is multi